docs/index.md

```diff
@@ -4,54 +4,31 @@ page_description: Introduces the Docker Registry
-is collection of images. Users interact with the registry by pushing images to
-or pulling images from the registry. The Docker Registry includes several
-optional features that you can configure according to your needs.
+A Registry is a storage and content delivery system, holding named Docker images, available in different tagged versions. For example, the image `distribution/registry`, with tags `2.0` and `latest`.
 
-![](images/registry.png)
+Users interact with a registry by using docker push and pull commands. For example, `docker pull myregistry.com/stevvooe/batman:voice`.
 
-The architecture supports a configurable storage backend. You can store images
-on  a file system or on a service such as Amazon S3 or Microsoft Azure. The
-default storage system is the local disk; this is suitable for development or
-some small deployments.
+Storage itself is delegated to drivers. The default storage driver is the local posix filesystem, which is suitable for development or small deployments. Additional cloud-based storage driver like S3 and Microsoft Azure are also supported. People looking into using other storage backends may do so by implementing their own driver following the [Storage API](storagedrivers.md).
 
-Securing access to images is a concern for even the simplest deployment. The
-registry service supports transport layer security (TLS) natively. You must
-configure it in your instance to make use of it. You can also use a proxy server
-such as Nginx and basic authentication to extend the security of a deployment.  
+Since securing access to your hosted images is paramount, the Registry does natively support TLS. You can also enforce basic authentication through a proxy like Nginx.
 
-The registry repository includes reference implementations for additional
-authentication and authorization support. Only very large or public registry
-deployments are expected to extend the registry in this way.
+The registry github repository includes reference implementations for additional authentication and authorization methods. Only very large or public registry deployments are expected to extend the registry in this way.
 
-Docker Registry architecture includes a robust notification system. This system
-sends webhook notifications in response to registry activity.  The registry also
-includes features for both logging and reporting as well. Reporting is useful
-for large installations that want to collect metrics. Currently, the feature
-supports both New Relic and Bugsnag.
+Finally, the Registry includes a robust [notification system](notifications.md), calling webhooks in response to activity, and both extensive logging and reporting. Reporting is mostly useful for large installations that want to collect metrics. Currently, New Relic and Bugsnag are supported.
 
 ## Getting help
 
-Docker Registry is an open source project and is under active development. If
-you need help, would like to contribute, or simply want to talk about the
```

Comment:
It *does* make it easier to add inline notes while reviewing on GitHub. Also, when reading the un-rendered Markdown (as plain-text), it's easier to read and better preserves the intent, irregardless of text-editor settings.

Maintaining the 80-chars wrap can be a PITA, so that's why we need to weigh the pros/cons of both and decide on a style-guide. Similar discussions (72-chars in that case) have been around formatting of commit-messages (http://stackoverflow.com/questions/2290016/git-commit-messages-50-72-formatting)